Config.cmake.in

```diff
@@ -0,0 +1 @@
+include("${CMAKE_CURRENT_LIST_DIR}/@targets_export_name@.cmake")
```

Comment:
Ill defer to you on the cmake config stuff, haven't really worked with it before using hunter.